packages/odyssey-react-mui/src/labs/DataTable.tsx

```diff
@@ -480,28 +501,139 @@ const DataTable = ({
           : undefined,
     }),
 
-    muiRowDragHandleProps: {
-      tabIndex: -1,
-      onDragEnd: () => {
-        const cols = table.getAllColumns();
-        cols[0].toggleVisibility();
-
-        const { draggingRow, hoveredRow } = table.getState();
-        if (draggingRow) {
-          handleReordering({
-            rowId: draggingRow.id,
-            newIndex: (hoveredRow as MRT_RowData).index,
-          });
-        }
+    muiRowDragHandleProps: ({ table, row }) => {
+      return {
+        title: "Drag row or press space/enter key to start and stop reordering",
+        ariaLabel:
+          "Drag row to reorder. Or, press space or enter to start and stop reordering and esc to cancel.",
+        onKeyDown: (event: KeyboardEvent<HTMLButtonElement>) => {
```

Comment:
I'd suggest extracting this out to its own function so that it's easier to read and the function itself isn't be recreated each time `muiRowDragHandleProps` is created